README.md

```diff
@@ -144,7 +144,10 @@ options|Options and add-ons for the given bundle. See `bundle.options` below for
 Property | Description | For Bundles | Default
 ---------|-------------|-------------|--------
 auth_n_authz|Accepts true/false. Enables Password Authentication and User Authorization.|Cassandra|false
-client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Spark|false
+client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Elasticsearch, Spark|false
+dedicated_master_nodes|Accepts true/false. Enables Dedicated Master Nodes.|Elasticsearch|false
+master_node_size|Desired master node sizes.|Elasticsearch|Required
+security_plugin|Accepts true/false. Enables Securiy Plugin.|Elasticsearch|false
```

Comment:
Typo in "Securiy". Also could this parameter be explained with a bit more detail?

Reply:
done